core/spree_core.gemspec

```diff
@@ -59,4 +59,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'friendly_id-mobility', '~> 1.0'
   s.add_dependency 'request_store', '~> 1.7'
   s.add_dependency 'wannabe_bool'
+  s.add_dependency 'geared_pagination'
```

Comment:
We need this in the featured taxon page section: https://github.com/spree/spree/pull/12317/files#diff-445e9722f0859b1a8bbee8a95b3e217d2a5aed7dd61fe9ee16b459c8783a0907R61